content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

```diff
@@ -101,6 +101,10 @@ The [Archive Service]({{< relref "../../../../tools/archiving-service/archiving-
 
 Corda now supports JDK Azul 8u382 and Oracle JDK 8u381.
 
+### Collaborative Recovery deprecated
+
+The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), was deprecated in Corda 4.11 and will be removed in Corda 4.12. You must now use the new recovery tools introduced in version 4.11, as detailed in the following 4.11 release notes.
```

Comment:
Slight update...

The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), is deprecated in Corda 4.11 and will be removed in Corda 4.12. You are advised to use the new recovery tools introduced in version 4.11, as detailed in the following 4.11 release notes.

Reply:
updated